assets/dev/scss/editor/navigator/navigator.scss

```diff
@@ -91,12 +93,7 @@ body {
 		}
 	}
 
-	&__elements.elementor-navigator-list__promotion {
-		height: calc(100% - 80px);
-	}
-
 	&__footer {
-		position: absolute;
 		width: 100%;
 		left: 0;
```

Comment:
Can we remove some of these properties, now that we have removed `position: absolute`?